deploy/Chart/templates/dashboard/rbac.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
 rules:
   - apiGroups: ['api.ucp.dev']
     resources: ['*']
-    verbs: ['get', 'list']
+    verbs: ['*']
```

Comment:
Could you add a comment here for context for anyone looking at this code later?

Reply:
added